nrf52-hal-common/src/spim.rs

```diff
@@ -56,6 +57,71 @@ impl_spim_ext!(
 /// - The over-read character is hardcoded to `0`.
 pub struct Spim<T>(T);
 
+impl<T> embedded_hal::blocking::spi::Transfer<u8> for Spim<T> where T: SpimExt
+{
+   type Error = Error;
+
+    fn transfer<'w>(&mut self, words: &'w mut [u8]) -> Result<&'w [u8], Error> {
```

Comment:
Why doesn't this method copy code to SRAM, if needed, just like `write` does?

Reply:
The buffer is mutable and will always be placed in data memory by the compiler, so there will never be the same need for a copy

Reply:
Ah, makes sense! Adding a comment that states just that would be great.